lib/bootscale/entry.rb

```diff
@@ -4,18 +4,20 @@ class Entry
       RbConfig::CONFIG['DLEXT'],
       RbConfig::CONFIG['DLEXT2'],
     ].reject { |ext| !ext || ext.empty? }.map { |ext| ".#{ext}"}
-    FEATURE_FILES = "**/*{#{DOT_RB},#{DL_EXTENSIONS.join(',')}}"
+    REQUIREABLE_FILES = "**/*{#{DOT_RB},#{DL_EXTENSIONS.join(',')}}"
     NORMALIZE_NATIVE_EXTENSIONS = !DL_EXTENSIONS.include?(DOT_SO)
     ALTERNATIVE_NATIVE_EXTENSIONS_PATTERN = /\.(o|bundle|dylib)\z/
+    SLASH = '/'.freeze
+    EMPTY = ''.freeze
 
     def initialize(path)
       @path = path
     end
 
-    def features
-      path_prefix = @path.end_with?('/'.freeze) ? @path : "#{@path}/"
-      Dir[File.join(@path, FEATURE_FILES)].map do |absolute_path|
-        relative_path = absolute_path.sub(path_prefix, ''.freeze)
+    def requireables
+      path_prefix = (@path.end_with?(SLASH) ? @path.size : @path.size + 1)
+      Dir[File.join(@path, REQUIREABLE_FILES)].map do |absolute_path|
+        relative_path = absolute_path.slice(path_prefix, 999)
```

Comment:
did not want to create the extra range on every file :)
created it outside the .map so that should be fine (450 vs 15k ranges created) ... but using 999 would be 0 new objects ... either way is fine though ...